src/internal/github/assets.go

```diff
@@ -1,14 +1,51 @@
 package github
 
 import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
 	"regexp"
+	httpInternal "registry-stable/internal/http"
 )
 
 type Platform struct {
 	OS   string
 	Arch string
 }
 
+func DownloadAssetContents(ctx context.Context, downloadURL string) (body io.ReadCloser, err error) {
+	httpClient := httpInternal.GetHttpRetryClient()
+
+	log.Printf("Downloading asset, url: %s", downloadURL)
```

Comment:
I've raised this for after we merge this PR: https://github.com/opentofu/registry-stable/issues/15

I think we should be using slog everywhere possible

Reply:
Yes I think we'll just have a cleanup task for that